delivery.module

```diff
@@ -188,10 +188,10 @@ function delivery_form_alter(&$form, FormStateInterface $formState, $formId) {
       /** @var \Drupal\workspaces\WorkspaceManagerInterface $workspaceManager */
       $workspaceManager = \Drupal::service('workspaces.manager');
       $workspace = $workspaceManager->getActiveWorkspace();
-      if ($defaultLanguage = $workspace->primary_language->value) {
+      if (!empty($workspace->primary_language) && $defaultLanguage = $workspace->primary_language->value) {
```

Comment:
@jamiehollern the `primary_language` and `secondary_languages` are only available if the submodule `delivery_allowed_languages` is enabled, otherwise this ends up in WSOD.